contracts/0.7.x/interfaces/IAppealEvents.sol

```diff
@@ -11,20 +11,28 @@ pragma solidity >=0.7;
 interface IAppealEvents {
 
     /** @dev To be emitted when the appeal fees of one of the parties are fully funded.
-     *  @param _itemID The ID of the respective transaction.
-     *  @param _ruling The ruling that is fully funded.
-     *  @param _round The appeal round fully funded by _party. Starts from 0.
+     *  @param _itemID The ID of the respective dispute.
+     *  @param _round The appeal round fully funded by _ruling. Starts from 0.
+     *  @param _ruling Indicates the ruling option which got fully funded.
      */
-    event HasPaidAppealFee(uint256 indexed _itemID, uint256 _ruling, uint256 _round);
+    event HasPaidAppealFee(uint256 indexed _itemID, uint256 _round, uint256 indexed _ruling);
 
     /**
      *  @dev To be emitted when someone contributes to the appeal process.
-     *  @param _itemID The ID of the respective transaction.
-     *  @param _ruling The ruling which received the contribution.
-     *  @param _contributor The address of the contributor.
+     *  @param _itemID The ID of the respective dispute.
      *  @param _round The appeal round to which the contribution is going. Starts from 0.
+     *  @param _ruling Indicates the ruling option which got the contribution.
+     *  @param _contributor Caller of fundAppeal function.
      *  @param _amount The amount contributed.
      */
-    event AppealContribution(uint256 indexed _itemID, uint256 _ruling, address indexed _contributor, uint256 _round, uint256 _amount);
+    event AppealContribution(uint256 indexed _itemID, uint256 _round, uint256 indexed _ruling, address indexed _contributor, uint256 _amount);
 
+    /** @dev To be raised inside withdrawFeesAndRewards function.
+     *  @param _itemID The ID of the respective dispute.
+     *  @param _round The appeal round to which the contribution is going. Starts from 0.
+     *  @param _ruling Indicates the ruling option which got the contribution.
+     *  @param _contributor Caller of fundAppeal function.
+     *  @param _reward Total amount of deposits reimbursed plus rewards.
+     */
+    event Withdrawal(uint256 indexed _itemID, uint256 indexed _round, uint256 _ruling, address indexed _contributor, uint256 _reward);
```

Comment:
This event should also be updated.

Reply:
Yeah it's done. I meant that I had changed the other events according to Withdrawal event

Reply:
itemID, round, ruling, contributor, value

Reply:
The only difference is that round is indexed in `Withdrawal` (I took that from the DisputeResolver interface) but ruling is indexed in `AppealContribution` and `HasPaidAppealFee`.

This is ok, right?

Reply:
Yeah, that's fine.
It's only a matter of not getting confused when reading data using numeric keys.
Most of the time we should stick to named parameters.